webblog/frontend/src/components/blog/BlogList.js

```diff
@@ -0,0 +1,81 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {deleteBlog, getBlogList} from "../../actions/blogList";
+
+export class BlogList extends Component {
+
+    static propTypes = {
+        blogList: PropTypes.array.isRequired,
+        getBlogList: PropTypes.func.isRequired,
+        deleteBlog: PropTypes.func.isRequired
+    };
+
+    state = {
+        currentPage: 1,
+        blogsPerPage: 3
```

Comment:
move these constants to environment variables file or constants files

Reply:
these are the initial states, they're updated for pagination.